components/URLs.js

```diff
@@ -130,7 +130,7 @@ const URLs = ({ onSubmit }) => {
                     {...field}
                     className='url-input'
                     icon={<MdDelete />}
-                    placeholder='https://'
+                    placeholder='eg: https://twitter.com/'
```

Comment:
I am wondering whether the presence of `e.g.,` could confuse rather than help? I would suggest to perhaps remove the `eg:` unless it has been used elsewhere in the same UI